doc/api/module.md

```diff
@@ -217,6 +217,69 @@ added: v22.8.0
 * Returns: {string|undefined} Path to the [module compile cache][] directory if it is enabled,
   or `undefined` otherwise.
 
+### `module.findPackageJSON(startLocation, parentLocation)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active Development
+
+* `startLocation` {string} Where to look (relative to `parentLocation`). This can be a
+  relative/unresolved specifier (ex `'..'`) or a package name.
+* `parentLocation` {string} The absolute location (file URL string or FS path) of the containing
```

Comment:
```suggestion
* `startLocation` {string|URL} Where to look (relative to `parentLocation`). This can be a
  relative/unresolved specifier (ex `'..'`) or a package name.
* `parentLocation` {string|URL} The absolute location (`file:` URL string or FS path) of the containing
```